lib/components/src/navigation/menu_link.js

```diff
@@ -0,0 +1,23 @@
+import glamorous from 'glamorous';
+import RoutedLink from './routed_link';
+
+export default glamorous(RoutedLink, { rootEl: 'a' })(
+  {
+    display: 'block',
+    color: '#828282',
```

Comment:
I guess as soon as it merged I can start updating and apodting #1360 for glamorous components. What do you think?